### DISCUSSION TITLE
Manually create an index on a field

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13292] [Archive]
As far as I can see, it's impossible to manually create an index on a field! This is vital for tables with more than a few hundred records!

**Comment by benhaynes** [comment]
Duplicate of https://github.com/directus/directus/discussions/5503

**Reply by sxsws** [reply]
it's going to have to be more than a toggle though - with the option to set an index name and also possible conditions for partial indexes.

**Reply by benhaynes** [reply]
Yeah, a full indexing system would be ideal, and there's a lot more thought that needs to go into that system to make it truly full functional.
